scripts/proposals/Wrapper.s.sol

```diff
@@ -124,6 +112,31 @@ contract Wrapper is Utils {
         }
     }
 
```

Comment:
It seems it is not rebased on the correct branch as this function already exists